### DISCUSSION TITLE
Anti-filtering detectors to prevent falsified photos and "flattering" images

### DISCUSSION THREAD

**Discussion by WilliamDormer** [PullDating/Org#29] [Ideas & Features]
A lot of people will enhance their photos before putting them on the dating app. Perhaps this could be resolved by forcing people to take the photos in app? But then they cannot show their best moments or hobbies. Alternatively, we could implement some sort of detection and removal through software for edited/enhanced photographs.

**Comment by aklidata** [comment]
I really like this idea!!

**Comment by Oshyan** [comment]
I love this in concept, however I do see some concerns as far as accuracy of the detection and outright disallowing the image on that basis. One example of where this might be an issue is that there are a fair number of phone cameras that auto-apply some kind of "smoothing" filter, etc., depending on mode. That said I think attempting to do this would be valuable, assuming the algo and compute time are both not burdensome to develop and run (probably run locally on the uploader's device).

In general I am interested in seeing features that encourage good behavior and discourage bad, but only truly enforcing things when it's necessary for the health of the platform and users. I think a good way to handle things like this (and perhaps many others) would be to simply provide a warning to both the uploader - "Your image appears to be potentially manipulated. It will display a small warning to viewers on your profile. Consider using another image, or provide an explanation in the caption if you'd prefer." - and to viewers, indicated by a small "!" icon perhaps, when clicked the text shows: "This user's image shows signs of being manipulated. You may want to keep this in mind when evaluating their profile." Something like that, anyway. Basically, give the user the info and let them decide what to do with it. All that has to be done in a way that's fair to both users, and unobtrusive but not obscured, obvious but not dominant, if that make sense (again a little colored ! icon or something could work).

**Reply by WilliamDormer** [reply]
I like the idea of indicating it to other users without outright banning them. And ya a little indicator on their public profile would be the perfect way to do that.

**Comment by ajbott** [comment]
What about "app verified photo", if the photo was taken in-app, then the photo would be tagged as a verified picture?
